daemon/migrations/20220404113202_create_closed_cfds_tables.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE IF NOT EXISTS closed_cfds (
+    id integer PRIMARY KEY autoincrement,
+    uuid text UNIQUE NOT NULL,
+    position text NOT NULL,
+    initial_price text NOT NULL,
+    taker_leverage integer NOT NULL,
+    n_contracts integer NOT NULL,
+    counterparty_network_identity text NOT NULL,
+    role text NOT NULL,
+    fees integer NOT NULL,
+    expiry_timestamp integer NOT NULL,
+    lock_txid text NOT NULL,
+    lock_dlc_vout integer NOT NULL
+);
+CREATE UNIQUE INDEX IF NOT EXISTS closed_cfds_uuid ON closed_cfds (uuid);
+CREATE TABLE IF NOT EXISTS collaborative_close_txs (
+    id integer PRIMARY KEY autoincrement,
+    cfd_id integer NOT NULL,
+    txid text NOT NULL,
+    vout integer NOT NULL,
+    payout integer NOT NULL,
+    price text NOT NULL,
+    FOREIGN KEY (cfd_id) REFERENCES closed_cfds (id)
```

Comment:
Thinking about it: we'd need to copy the timestamp from the `event` table.

Reply:
I think you're thinking about debugging, since I don't think this information is needed in `projection`. I see how it could be useful, although it's fairly easy to take the txid and use a block explorer.

> Thinking about it: we'd need to copy the timestamp from the event table.

Incidentally, that timestamp reflects when our system noticed that a particular transaction got sufficient confirmations to consider it final. Is that what you would like to have when debugging?

Reply:
I've added this in a fixup commit!